riak_test/log_handoff_test.erl

```diff
@@ -75,6 +75,7 @@ test_handoff(RootNode, NewNode, NTestItems) ->
     %% added to the root node.  Note: systest_read() returns
     %% /non-matching/ items, so getting nothing back is good:
     lager:info("Validating data after handoff:"),
+    timer:sleep(100),
```

Comment:
No, not even once.

This will just not do, as it will ultimately result in a non-deterministic test.